server/src/routes/posts.js

```diff
@@ -30,8 +30,13 @@ router.post('/', async (req, res) => {
 
 // Update one post
 // eslint-disable-next-line no-unused-vars
-router.patch('/:id', (req, res) => {
-  // TODO
+router.patch('/:id', async (req, res) => {
+  try {
+    await Post.update({ _id: req.params.id }, { title: req.body.title, body: req.body.body });
+    res.status(200).json({ message: 'ok' });
```

Comment:
Probably no need to have an 'ok' message here as it is already reflected in the HTTP status code.